python/dgl/sparse/matmul.py

```diff
@@ -17,14 +17,14 @@ def spmm(A: Union[SparseMatrix, DiagMatrix], X: torch.Tensor) -> torch.Tensor:
     Parameters
     ----------
     A : SparseMatrix or DiagMatrix
-        Sparse matrix of shape (N, M) with values of shape (nnz)
+        Sparse matrix of shape ``(L, M)`` with scalar values
     X : torch.Tensor
-        Dense tensor of shape (M, F) or (M)
+        Dense matrix of shape ``(M, N)`` or ``(M)``
```

Comment:
Is there any convention on why picking these alphabets?

Reply:
Not I am aware of any global convention.
I changed this to keep everything consistent in DGL sparse tutorial and doc.